.gitmodules

```diff
@@ -4,4 +4,4 @@
 [submodule "lib/colonyToken"]
 	path = lib/colonyToken
 	url = https://github.com/JoinColony/colonyToken.git
-  branch = master
+  branch = maintenance/upgrades
```

Comment:
Note that this will be switched back to `master` when we merge the changes in the `colonyToken` repo